cli/operatorsdk.go

```diff
@@ -28,10 +28,11 @@ type OperatorSdkScorecardResult struct {
 }
 
 type OperatorSdkBundleValidateOptions struct {
-	LogLevel         string
-	ContainerEnginer string
-	Selector         []string
-	OutputFormat     string
+	LogLevel        string
+	ContainerEngine string
```

Comment:
Just want to confirm: this is here as a pass-through flag for operator-sdk, correct? We aren't exposing this to the user in any way, so this is _technically_ not configurable today, but may be something we can configure in the future?

Reply:
Correct. Not configurable by the user. It may not be necessary if we never actually support docker instead of podman. Also, if we move to API calls instead of Exec calls, this would probably be moot.